pulse2percept/models/granley2021.py

```diff
@@ -90,6 +91,27 @@ def __call__(self, freq, amp, pdur):
         F_bright = self.predict_freq_amp(amp * self.scale_threshold(pdur), freq)
         return F_bright
 
+    def __eq__(self, other):
```

Comment:
This also seems like one `__eq__` function in BaseModel could cover this and all of the other effect models